test/util.ts

```diff
@@ -39,7 +39,7 @@ export function getTestStore(): Store {
         text: ['Rickrolling'],
       },
       captchaHandler: {
-        challenge: 'img.thumbimage',
+        challenge: '#mw-content-text > div.mw-content-ltr.mw-parser-output > figure:nth-child(3) > a > img',
```

Comment:
This type of selector can be inconsistent, but thank you for updating and bringing it to my attention.

Try to aim for a specific selector (like the previous one) to avoid unintended interactions with other elements on the page.

```suggestion
        challenge: '.mw-file-element',
```